Sources/FluentSQL/SQL+QuerySupporting.swift

```diff
@@ -158,7 +158,7 @@ extension QuerySupporting where
     QueryKey.Expression == QueryKey.Expression.Function.Argument.Expression
 {
     /// See `QuerySupporting`.
-    public static func queryAggregate(_ name: QueryAggregate, _ fields: [QueryKey]) -> QueryKey {
+    public static func queryAggregate(_ name: QueryAggregate, _ fields: [QueryKey], _ identifier: String) -> QueryKey {
```

Comment:
reason I created a new method is that default values can not be protocolised in the future, happy to change if needed